### ISSUE TITLE
[bug]: Extremely slow generation issues from installing xformers due to versioning conflicts with torch2.0

### ISSUE THREAD

**Issue by arigbs** [invoke-ai/InvokeAI#3348]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS

Linux

### GPU

cuda

### What version did you experience this issue on?

2.3.5

### What happened?

Updating to 2.3.5 with xformers installed, while successful, results in incredibly slow generations. Seeing this is caused by incompatiblity with the torch version installed, updating torch gets torch2.0 installed all the time, which also throws incompatiblity issues, stopping invoke from running. 

Last attempt, from running: 
`pip install torch torchvision --force-reinstall --extra-index-url https://download.pytorch.org/whl/cu117
pip install xformers`  

(as directed here: https://invoke-ai.github.io/InvokeAI/installation/010_INSTALL_AUTOMATED/#package-dependency-conflicts) leads top invoke failing during generation with the following in the console: "AttributeError: module 'xformers.ops' has no attribute 'memory_efficient_attention'", yet this: " xformers memory-efficient attention is available and enabled" appears while launching the invoke web ui, after getting the following immediately after running 
pip install xformers: 

ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
xformers 0.0.16 requires torch==1.13.1, but you have torch 2.0.1+cu117 which is incompatible.
numba 0.56.4 requires numpy<1.24,>=1.18, but you have numpy 1.24.3 which is incompatible.
invokeai 2.3.5 requires numpy<1.24, but you have numpy 1.24.3 which is incompatible.
invokeai 2.3.5 requires requests==2.28.2, but you have requests 2.30.0 which is incompatible.
invokeai 2.3.5 requires torch~=1.13.1, but you have torch 2.0.1+cu117 which is incompatible.
aiohttp 3.8.3 requires charset-normalizer<3.0,>=2.0, but you have charset-normalizer 3.1.0 which is incompatible.

The issue is particularly hard to follow up on because each attempt to reinstall torch comes at the cost of almost 2gb of data and hours of waiting.

**Comment by gogurtenjoyer** [comment]
I suspect this can happen if you upgrade from an older version of InvokeAI that doesn't have knowledge of newer diffusers/torch/etc library versions that have come out since. I would try renaming your current 'invokeai' folder to `invokeai-old` and then installing a fresh copy of 2.3.5 (delete the unzipped installer zip, unzip the zip again, etc). Then you can move your models, configs, and outputs over from the old folder.

**Comment by shawarden** [comment]
The 'latest' version InvokeAI, v2.3.5 requires torch~=1.13.1

**Comment by MohammadAboulEla** [comment]
I'm using invoke v3.7.0 and I just noticed that updating pytorch and uninstalling xformers completely gives me more speed and better performance
